recipe/meta.yaml

```
@@ -39,7 +38,7 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.41.*
+    - llvmlite 0.42.0rc1
```

Think this would still work (even with an RC). Though could make sense to keep it tighter if things change significantly between RCs or the final release

```suggestion
    - llvmlite 0.42.*
```